discovery-provider/alembic/versions/3403e008e6c7_repair_missing_wallets.py

```diff
@@ -0,0 +1,51 @@
+"""repair-missing-wallets
+
+Revision ID: 3403e008e6c7
+Revises: 9931f7fd118f
+Create Date: 2022-08-08 19:45:39.391707
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "3403e008e6c7"
+down_revision = "9931f7fd118f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute(
+        """
+      begin;
+
+      update associated_wallets
+      set is_delete = false
+      where
+        is_delete = true and
+        is_current = true and
+        -- July-14-2022 03:15:40 PM +-7 UTC
+        blocknumber > 28370000 and
+        -- August-08-2022 12:44:00 PM +-7 UTC
+        blocknumber < 28754688 and
```

Comment:
Sanity checking - is there any case where there's an is_current = true after the aug 8th block, but is_delete is still false and we fail to update it? (seems very unlikely)

Reply:
it shouldn't be possible, but i want to make sure all discovery nodes are the same and do the right thing when they pick up the change, so we really do need an "end block"